jobs/e2e_node/crio/crio_cgroupsv2_imagefs.ign

```diff
@@ -8,6 +8,24 @@
     ]
   },
   "storage": {
+    "disks": [
+      {
+        "device": "/dev/disk/by-id/coreos-boot-disk",
+        "partitions": [
+          {
+            "label": "root",
+            "number": 4,
+            "resize": true,
+            "sizeMiB": 8192
+          },
+          {
+            "label": "imagefs",
+            "sizeMiB": 0
```

Comment:
I had no idea you could add a partition with a 0-byte size.

Reply:
I don't think that is what this means. Its means that we use the rest of the filesystem for imagefs.

https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_setting_up_separate_var_mounts

Reply:
Indeed! Found details here too:

- [Sizing the root partition](https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_sizing_the_root_partition)

Thank you! :+1: